pydantic/validate_call_decorator.py

```diff
@@ -16,6 +17,50 @@
     AnyCallableT = TypeVar('AnyCallableT', bound=Callable[..., Any])
 
 
+def _check_function_type(function: object):
+    ERROR_CODE: PydanticErrorCodes = 'validate-call-type'
```

Comment:
```suggestion
    ERROR_CODE = 'validate-call-type'
```

No need to explicitly annotate here